ray_julia_jll/gen/artifacts.jl

```diff
@@ -60,8 +60,5 @@ if abspath(PROGRAM_FILE) == @__FILE__
             download_info=[(artifact_url, sha256sum(artifact_path))],
             force=true
         )
-
-        host_wrapper = joinpath(WRAPPERS_DIR, "$platform_str.jl")
-        cp("wrapper.jl.tmp", host_wrapper; force=true)
```

Comment:
We should probably run these scripts as part of CI just to validate they aren't broken

Reply:
~the template was deleted though in an earlier refactor?~

oh do you mean run `make.jl` etc? yeah we can do that!